sources/System/Renderer/RenderSystem.hpp

```diff
@@ -21,6 +21,13 @@ namespace BBM
 
 		//! @brief The camera used to render.
 		RAY::Camera::Camera3D _camera;
+
+		//! @brief Defines if the debug informations must be displayed or not
+		bool _debugMode;
+
+		//! @brief Window framerate limit
+		static constexpr short FPS = 30;
```

Comment:
didn't we say it was 60 ?

Reply:
why does the debug set the framerate ?
your debug should only display the calculated framerate.